acme_diags/acme_diags_driver.py

```diff
@@ -28,9 +28,6 @@
 from acme_diags.parser.core_parser import CoreParser
 from acme_diags.viewer.main import create_viewer
 
-# turn off MPI in cdms2 -- not currently supported by e3sm_diags
-cdms2.tvariable.HAVE_MPI = False
```

Comment:
Setting the `HAVE_MPI` is no longer necessary since `os.environ["CDMS_NO_MPI"]` is now used.